### ISSUE TITLE
Input Settings>Capture system keyboard shortcuts option missing: 

### ISSUE THREAD

**Issue by ashleyw-gh** [moonlight-stream/moonlight-android#1281]
### Is your feature request related to a problem? Please describe.

On windows Moonlight client, there is an option under the "Input Settings" for "Capture system keyboard shortcuts"
That option doesn't exist on Moonlight client for Android.

### Describe the solution you'd like

Please can the "Capture system keyboard shortcuts" be implemented on the Android TV moonlight client.
*We have a common use case of needing to pass-through <windows key> combinations from a USB keyboard connected to the ShieldTV.

### Describe alternatives you've considered

Adding the USB keyboard directly to the Game PC server doesn't work in our case as the main Game PC server is in a different room to where we need to use the keyboard.

### Screenshots

![windowsclientoption](https://github.com/moonlight-stream/moonlight-android/assets/11339367/a3522e58-a962-44e5-b5ec-2d1d2add6e6c)

**Comment by cgutman** [comment]
Moonlight does capture keyboard shortcuts using Samsung's vendor-specific API. The problem is that Android itself intentionally provides no way to capture system-wide keyboard shortcuts like PC platforms do.

**Comment by ashleyw-gh** [comment]
thanks, so that means its a no go to get things like <alt><tab> and <windowskey>combinations working from the android client?
Then I'll need to explore possibilities but other alternatives such as USB over IP type solutions just complicate my setup sadly, so my options are limited.

**Comment by cgutman** [comment]
Yes, I believe so. I doubt you will find an app that can directly intercept system keyboard shortcuts, except maybe apps that act as an IME (software keyboard). However, IMEs are highly scrutinized by Google due to the risk of privacy issues, and making Moonlight act like an IME would be against Play Store policy (since it's not core functionality of the app).

An app that takes full control of the USB device itself _might_ work, but I seem to recall Google restricts certain classes of USB devices from direct access for security/privacy reasons. An app with direct access to keyboard devices could perform covert keylogging. Howver, this approach wouldn't be feasible as a general method to support keyboard capture because it would require separate keyboard drivers built for all possible buses that could be used to attach a keyboard, and some of those don't have Android APIs to allow custom drivers to be written like USB and Bluetooth do.
